### PR TITLE
issue #1124

### PR DESCRIPTION
**Pull request by ghost**
Updated **mq.scss**

### CODE REVIEW

**Comment by ghost** [comment]
Hello, Is there anyone who can merge the changes?? I pulled request on 26 Jan

**Comment by ghost** [comment]
Isn't there anyone who can merge the changes??

File: static/sass/mq.scss [hunk 1]
```diff
@@ -159,4 +159,48 @@
  * for IE10 Snap Mode on Metro
  */
 @-ms-viewport { width: device-width; }
-@viewport { width: device-width; }
\ No newline at end of file
+@viewport { width: device-width; }
+
+
+
+@media(min-width: px2em(940px)) {
```
Comment (berkerpeksag) [inline static/sass/mq.scss]: Style nit: Please add spaces around `XXpx`: `px2em(940px)` -> `px2em( 940px )`

File: static/sass/mq.scss [hunk 2]
```diff
@@ -159,4 +159,48 @@
  * for IE10 Snap Mode on Metro
  */
 @-ms-viewport { width: device-width; }
-@viewport { width: device-width; }
\ No newline at end of file
+@viewport { width: device-width; }
+
+
```
Comment (berkerpeksag) [inline static/sass/mq.scss:164]: Style nit: Please add two new lines instead of three.

File: static/sass/mq.scss [hunk 3]
```diff
@@ -159,4 +159,48 @@
...
+
+@media(min-width: px2em(940px)) {
+  #mainnav .navigation.menu .tier-1 {
+    &.element-2 .subnav {
+      overflow: hidden;
+      overflow-y: scroll;
+      height: px2em(180px);
```
Comment (berkerpeksag) [inline static/sass/mq.scss]: I'm still not sure setting `height` to a fixed value is a good idea here. Don't we still see the original bug if we visit the website in a 1280x150px window?

**Review by berkerpeksag — COMMENTED** [review]
Please regenerate CSS files by using the command at https://pythondotorg.readthedocs.io/install.html#generating-css-files-automatically

File: static/sass/mq.scss (hunk 3 above)
Comment (ghost) [reply static/sass/mq.scss]: Yeah, Now it won't show original bug even the height is less than **180PX**.

**Pull request closed without merging**